x-pack/plugins/infra/public/containers/metrics_source/source_errors.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/* eslint-disable max-classes-per-file */
+
+import { HttpFetchError } from '@kbn/core-http-browser-internal/src/http_fetch_error';
```

Comment:
Nit: If this is only used as a type you might wanna mark it as a type import. I don't know if our tools care though in the end.